pages/index.js

```diff
@@ -8,12 +9,18 @@ import { queries, withApiData } from "../api";
 import { getTeamScore } from "@/lib/getTeamScore";
 import DefaultPicture from "@/public/default-profile.jpeg";
 
+const sortByScore = sortWith([descend(prop("score"))]);
+
 function Home({ data, refetch }) {
   const orderedTeams = useMemo(() => {
-    if (data?.allTeams?.data) {
-      return data.allTeams.data;
+    if (!data?.allTeams?.data) {
+      return [];
     }
-    return [];
+    return sortByScore(
+      [...data.allTeams.data].map((team) => {
```

Comment:
```suggestion
      data.allTeams.data.map((team) => {
```

if you want to have a pure ramda solution to the whole orderedTeams:

```
const getOrderedTeams = pipe(
  pathOr([], ['allTeams', 'data']),
  map(team => ({ ...team, score: getTeamScore(team) }), 
  sortWith([descend(prop("score"))])
)
```